db/seeds.rb

```diff
@@ -11,7 +11,7 @@
 
 number_of_ads.times do
   ads_data << {
-    city: %w[Slatina Neum Makarska Varaždin Varazdin Bibinje Zagreb Križevci Krizevci].sample,
+    city: City.find(rand(1..(City.count-1))),
```

Comment:
`rand(1..(City.count-1))` can easily return city that is not in a database. It will also run find + count for each iteration. Here is a suggestion.

```ruby
cities = City.limit(number_of_ads)
number_of_ads.times do
  ads_data << {
    city: City.find(cities.sample)
```

Reply:
...or how about this:

```suggestion
cities = City.pluck(:id)
# ...
    city_id: cities.sample,
```

?

Reply:
Even better 👍